docs/api/graphql_queries.md

```diff
@@ -216,18 +240,54 @@ Response:
 
 ## Querying Locations
 
-!!! tip
+You can get the Location object from any item by querying for `_location` or `_allLocations`.
+When you use `_location`, the API returns:
```

Comment:
We may also want to document the `_url` field, that returns an URL alias for the content item. It behaves like `_location`, and guesses the "best" location based on a set of rules (it is one of the core features of that change).